src/test/java/com/hocs/test/glue/decs/BaseStepDefs.java

```diff
@@ -82,7 +80,7 @@ public void iSelectSaveChanges() {
     public void theCaseShouldBeClosed() {
         dashboard.getCurrentCase();
         caseView.assertCaseCannotBeAllocated();
-        summaryTab.assertThereIsNoActiveStage();
+        summaryTab.asserNoActiveStageVisible();
```

Comment:
Seem to have removed a 't' from this, the base method must have been incorrectly renamed because this would've been flagged

Reply:
fixed